javascript/javascript1/week4/voice_assistant.js

```diff
@@ -0,0 +1,131 @@
+let username = "";
+let todo = [];
+
+function help() {
+  const help = {
+    "Hello my name is *your name*": "Saves your name in the system",
+    "What is my name": "Tells you your name",
+    "Add *something* to my todo": "Adds *something* to your todo list",
+    "Remove *something* from my todo":
+      "Removes *something* from your todo list ",
+    "What is on my todo?": "Gives you the whole todo list",
+    "What day is it today?": "Tells you the current date",
+    "What is *math operation*": "Solves math operation(fx 1+2)",
+    "Set a timer for x minutes/seconds/hours":
+      "Sets a timer for x minutes/seconds/hours",
+    help: "Shows this list",
+    "Forget my name": "Removes your name from the system",
+  };
+  for (const [key, value] of Object.entries(help)) {
+    console.log(`${key}: ${value}`);
+  }
+}
+function addToDo(activity) {
+  todo.push(activity);
+}
+
+function removeFromToDo(activity) {
+  let index = todo.indexOf(activity);
+  if (index > -1) {
+    todo.splice(index, 1);
+  }
+}
+
+function showToDos() {
+  const message = "You have the following todos:";
+  let toDoCount = ``;
+  for (let i = 0; i < todo.length; i++) {
+    toDoCount += `\n${i + 1}. ${todo[i]}\n`;
+  }
+  return message + toDoCount;
```

Comment:
Clever! I'd rename `toDoCount` to `toDoList`. `*count` is more appropriate to numbers